src/Credentials/CredentialProvider.php

```diff
@@ -47,12 +47,15 @@ class CredentialProvider
     const ENV_SESSION = 'AWS_SESSION_TOKEN';
     const ENV_PROFILE = 'AWS_PROFILE';
 
+    use CredentialTrait;
+
     /**
      * Create a default credential provider that first checks for environment
      * variables, then checks for the "default" profile in ~/.aws/credentials,
-     * then tries to make GET Request to fetch credentials if Ecs environment
-     * variable is presented, and finally checks for EC2 instance profile
-     * credentials.
+     * then checks for "profile default" profile in ~/.aws/config,
```

Comment:
Can you document what this is (e.g., the default profile in the AWS CLI)? There's some documentation about the default credential chain that I think needs to be updated to reflect this change as well.